iothub/device/samples/PnpDeviceSamples/Thermostat/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Shared;
+using PnpHelpers;
+
+namespace TemperatureController
+{
+    public class Program
+    {
+        private const string DeviceConnectionString = "device_connection_string_here";
+        private const string ModelId = "dtmi:com:example:TemperatureController;1";
+        private const string Thermostat1 = "thermostat1";
+        private const string Thermostat2 = "thermostat2";
+
+        private static DeviceClient s_deviceClient;
+
+        public static async Task Main(string[] _)
+        {
+            await RunSampleAsync();
+        }
+
+        private static async Task RunSampleAsync()
+        {
+            // This sample follows the following workflow:
+            // -> Initialize device client instance.
+            // -> Send "current temperature" over both telemetry and property updates.
```

Comment:
Does this also need to add "Register callback and Handle command (reboot)"?

Reply:
yes, this is the initial impl, the commands and desired property callback implementations will follow

Reply:
Got it.